src/multio/fdb5/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ ecbuild_add_library(
 
     TARGET multio-fdb5
 
+    TYPE SHARED # Due to reliance on factory self registration this library cannot be static
```

Comment:
Does this mean this line should be added to all self-registering libraries, such as multio-maestro, etc?

Reply:
Indeed. I just fixed what was needed for getting ifs-bundle to compile and run successfully with `-DBUILD_SHARED_LIBS=OFF`. Only fdb5 is used there. If you have more where this applies, go ahead and add to this PR. You should have write-access to my fork's branch. Note that `multio` itself seems to be OK as static; at least what I observed.

Reply:
ping